javascript/src/main/pages/Home/Home.js

```diff
@@ -37,7 +37,7 @@ const Home = () => {
             <div className="text-left">
                 <h5>Welcome to the UCSB Courses Search App!</h5>
                 <BasicCourseSearchForm setCourseJSON={setCourseJSON} fetchJSON={fetchBasicCourseJSON} />
-                <Button><CSVLink style={{color: "white"}} headers={courseHeaders} data={courseJSON.classes} filename = {"CourseTable.csv"}>Download CSV</CSVLink></Button>
+                <Button style={{margin: "1rem 0"}}><CSVLink style={{color: "white"}} headers={courseHeaders} data={courseJSON.classes} filename = {"CourseTable.csv"}>Download CSV</CSVLink></Button>
```

Comment:
One small nitpick - this line is getting somewhat long with this change, so just like how the children of other components have their own lines and are tabbed in, can we move the `CSVLink` component to its own line? 

Something like this:
```
<parent ...>
    <child ...></child>
</parent>
```

Maybe you can even go as far as moving the props into their own line as well, if it's still too long. There should be some examples in the codebase of this.

Reply:
Andrew, I fixed the line with the latest commit. If you could take a look again that would be great.